2.train-VAE/3.beta_vae_training_extract_model_components.ipynb

```diff
@@ -9,7 +9,7 @@
      "name": "stderr",
```

Comment:
<p>Using the power of a notebook here is great. I like to treat notebooks like a laboratory notebook. I try to add comments explaining what a figure is, what the interpretation is and maybe why its useful or the significance. ~3 sentences. A small blurb explaining what KL is might help a new reader on this project such as myself.</p><p> </p><p>Also consider double checking the testing vs validation data, it looks like you have a testing subset of data used above. Maybe check with greg if this is validation or testing data. The whole testing vs validation thing gets confusing for me, happy to talk through in person if wanted. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>